apps/api/prisma/schema.prisma

```diff
@@ -316,8 +316,45 @@ model Document {
   redacted             Boolean  @default(false)
   isDeleted            Boolean  @default(false)
   createdAt            DateTime @default(now())
-  fileSize             Int      @default(0)
+  fileSize             Int?     @default(0)
```

Comment:
if we are making fileSize nullable, do we need the default value now?  (I only added the default value 0 in my original version of the Document table, to avoid having a nullable field)